Holovibes/sources/core/pipe.cc

```diff
@@ -500,7 +440,7 @@ void Pipe::insert_filter2d_view()
     if (api::get_filter2d_enabled() && api::get_filter2d_view_enabled())
     {
         fn_compute_vect_.conditional_push_back(
-            [&]()
+            [this]()
```

Comment:
what does this change ? (im an idiot)

Reply:
`[&]` catch all variables in the scope (so variable in function, function parameters, class member and this). While `[this]` only catch a reference to `this`.
In practice the compiler can sometimes replace `[&]` by the exact list but `[this]` is explicit